CMake Error - C2137
https://msdn.microsoft.com/ru-ru/library/fhxwbt0t.aspx?f=255&MSPPError=-2147217396

`edit vcpkg\buildtrees\libkml\src\libkml-1.3.0\src\kml\base\file_win32.cc`
45 str - no `''`, yes `' '` ;))
  `std::wstring wstr(str.length(), L'');` ->   `std::wstring wstr(str.length(), L' ');`

--- a/ports/libkml/patch_empty_literal_on_vc.patch
+++ b/ports/libkml/patch_empty_literal_on_vc.patch
@@ -1,0 +1,13 @@
+diff --git a/src/kml/base/file_win32.cc b/src/kml/base/file_win32.cc
+index c46e099..28ccb36 100644
+--- a/src/kml/base/file_win32.cc
++++ b/src/kml/base/file_win32.cc
+@@ -42,7 +42,7 @@ namespace kmlbase {
+ // Internal to the win32 file class. We need a conversion from string to
+ // LPCWSTR.
+ static std::wstring Str2Wstr(const string& str) {
+-  std::wstring wstr(str.length(), L'');
++  std::wstring wstr(str.length(), L' ');
+   std::copy(str.begin(), str.end(), wstr.begin());
+   return wstr;
+ }

--- a/ports/libkml/portfile.cmake
+++ b/ports/libkml/portfile.cmake
@@ -7,6 +7,13 @@ vcpkg_from_github(
     HEAD_REF master
 )
 
+vcpkg_apply_patches(
+    SOURCE_PATH ${SOURCE_PATH}
+    PATCHES
+    "${CMAKE_CURRENT_LIST_DIR}/patch_empty_literal_on_vc.patch"
+)
+
+
 vcpkg_configure_cmake(
     SOURCE_PATH ${SOURCE_PATH}
 )

